docs/18-guides/99-contribution/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Contributing to Nordcraft Documentation
+title: Contributing
 description: Learn how to create well-structured, consistent documentation pages for Nordcraft, following formatting guidelines and content best practices.
 ---
 
```

Comment:
Let's change the H1 of the main page on line 6 to "Contributing" as well as per the style guidelines.